src/io/flutter/dart/DartPlugin.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.dart;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.openapi.util.Version;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Provides access to the Dart Plugin.
+ */
+public class DartPlugin {
+
+  /**
+   * Tracks the minimum required Dart Plugin version.
+   */
+  private static final String MINIMUM_REQUIRED_PLUGIN_VERSION = "162.2233";
```

Comment:
You mean it should be in the Dart Plugin?

My reasoning is that it captures the Flutter Plugin's expectations about the Dart Plugin so it makes more sense for it to be there.  In an ideal world this would be in the `plugin.xml` but IDEA doesn't support version constraints.

Or maybe I'm missing your thrust?